apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,399 @@
+  alternative_group_number: z.number().nullable(),
+  alternative_group_size: z.number(),
+  assessment_question_advance_score_perc: z.number().nullable(),
+  avg_question_score_perc: z.number().nullable(),
+  display_name: z.string().nullable(),
+  number: z.string().nullable(),
+  open_issue_count: z.string().nullable(),
+  other_assessments: z
+    .array(
+      z.object({
+        color: z.string(),
+        label: z.string(),
+        assessment_id: IdSchema,
+        course_instance_id: IdSchema,
+      }),
+    )
+    .nullable(),
+  sync_errors_ansified: z.string().optional(),
+  sync_errors: z.string().nullable(),
+  sync_warnings_ansified: z.string().optional(),
+  sync_warnings: z.string().nullable(),
+  topic: TopicSchema.nullable(),
+  qid: z.string(),
+  start_new_zone: z.boolean().nullable(),
+  start_new_alternative_group: z.boolean().nullable(),
+  tags: z
+    .array(
+      z.object({
+        color: z.string(),
+        id: IdSchema,
+        name: z.string(),
+      }),
+    )
+    .nullable(),
+  title: z.string().nullable(),
+  zone_best_questions: z.number().nullable(),
+  zone_has_best_questions: z.boolean().nullable(),
+  zone_has_max_points: z.boolean().nullable(),
+  zone_max_points: z.number().nullable(),
+  zone_number_choose: z.number().nullable(),
+  zone_number: z.number().nullable(),
+  zone_title: z.string().nullable(),
+});
+type AssessmentQuestionRow = z.infer<typeof AssessmentQuestionRowSchema>;
+
+export function InstructorAssessmentQuestions({
+  resLocals,
+  questions,
+}: {
+  resLocals: Record<string, any>;
+  questions: AssessmentQuestionRow[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", resLocals)}
+        <script src="${nodeModulesAssetPath('lodash/lodash.min.js')}"></script>
+        <script src="${nodeModulesAssetPath('d3/dist/d3.min.js')}"></script>
+        <script src="${assetPath('localscripts/histmini.js')}"></script>
+        ${compiledScriptTag('instructorAssessmentQuestionsClient.ts')}
+      </head>
+      <script></script>
+      <body>
+        <script>
+          $(() => {
+            $('[data-toggle="popover"]').popover({ sanitize: false });
+
+            $('.js-sync-popover[data-toggle="popover"]')
+              .popover({
+                sanitize: false,
+              })
+              .on('show.bs.popover', function () {
+                $($(this).data('bs.popover').getTipElement()).css('max-width', '80%');
+              });
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          <div
+            class="modal fade"
+            id="resetQuestionVariantsModal"
+            tabindex="-1"
+            role="dialog"
+            aria-labelledby="resetQuestionVariantsModalLabel"
+          >
```

Comment:
Let's use the `Modal` component here.

Reply:
Updated this to a `Modal` component in [d833d29](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/d833d29e6af8ffd2dbe97453d413dfd718f7c289).